### ISSUE TITLE
I can't upload my project correctly

### ISSUE THREAD

**Issue by Gena-developer** [Autodesk-Forge/forge-configurator-inventor#484]
Hi, maybe someone will help me, I can't figure out what's going on. I always have empty in the drawing 
![image](https://github.com/Autodesk-Forge/forge-configurator-inventor/assets/114415779/50b0e2d0-5733-48f5-ad78-e9b91944ddb1)

Should be like this:
![image](https://github.com/Autodesk-Forge/forge-configurator-inventor/assets/114415779/f6850ac9-18f4-46f9-8ee4-c45555d3e0e0)

This is my project:
https://drive.google.com/file/d/1v5wUGsgbZ6PmcdXlPNdssn459CsGOgHp/view?usp=sharing
Log:
[Reports.zip](https://github.com/Autodesk-Forge/forge-configurator-inventor/files/12083000/Reports.zip)

**Comment by Gena-developer** [comment]
Hi, I saw the log file in other tabs, but here is empty.
![image](https://github.com/Autodesk-Forge/forge-configurator-inventor/assets/114415779/7cfedcab-0565-4bdc-9921-e0f653d62682)

**Comment by mivasi** [comment]
> Hi, I saw the log file in other tabs, but here is empty.
The reason is, it is using cached results from previous processing. If you want it to process again, you have to delete the cache, or inside the model properties choose different values, for which processing of drawings never happens.

**Comment by Gena-developer** [comment]
I got it
![image](https://github.com/Autodesk-Forge/forge-configurator-inventor/assets/114415779/66e1a73d-6fcb-4fe2-b2e8-390157ca1780)

[report (4).txt](https://github.com/Autodesk-Forge/forge-configurator-inventor/files/12111780/report.4.txt)

**Comment by mivasi** [comment]
In the picture showing configurator in browse I see you are trying to see Drawing to generate PDF: Templates\English\ANSI (in).dwg, but in Inventor you show different drawing file "14-00126.dwg". Can you see and select on left side inside the webapp drawing named 14-00126.dwg?

**Comment by Gena-developer** [comment]
No, I can't choose it, I don't know why, it exists in the project

**Comment by Gena-developer** [comment]
This is a problem, I can't view my file on the right side

**Comment by mivasi** [comment]
I will need some time to take your design and try it out.

**Comment by Gena-developer** [comment]
If you can do this, I will be very grateful to you, otherwise I'm just stuck.

**Comment by mivasi** [comment]
I would say you just selected wrong drawing in the left panel. If you select the first one called 14-00126.idw. See my screenshot.

 
![Screenshot 2023-07-22 224350](https://github.com/Autodesk-Forge/forge-configurator-inventor/assets/344539/3e2a3ea1-47b9-4c57-8c6f-bb6fce517b6b)

**Comment by Gena-developer** [comment]
I have done it! Thank for all!

**Comment by Gena-developer** [comment]
Hi, please tell me how can I find what the problem is? I have a log file ,and error of application. I cannot upload a next project.

![image](https://github.com/Autodesk-Forge/forge-configurator-inventor/assets/114415779/cc6704bf-8090-4a70-8cd1-94a459328186)

[2023-08-09_18-13-50_ac78199448c2493181a6258c7cc6cf48.txt](https://github.com/Autodesk-Forge/forge-configurator-inventor/files/12305771/2023-08-09_18-13-50_ac78199448c2493181a6258c7cc6cf48.txt)
